static/app/components/events/featureFlags/eventFeatureFlagList.tsx

```diff
@@ -91,8 +91,8 @@ export function EventFeatureFlagList({
     event,
   });
 
-  const hasFlagContext = !!event.contexts.flags;
-  const hasFlags = Boolean(hasFlagContext && event?.contexts?.flags?.values.length);
+  const hasFlagContext = !!event.contexts?.flags?.values;
```

Comment:
i'd look at why typescript didn't throw an error and fix the types

Reply:
because we currently require `flags.values` to exist on the event type :( didn't expect people to override it

Reply:
if the user can override it then we'll probably want some function that does light validation on the shape you expect

Reply:
updated the type to make `values` optional as per @ryan953 's suggestion